tape/src/test/java/com/squareup/tape/QueueTestUtils.java

```diff
@@ -19,7 +19,9 @@
   static File copyTestFile(String file) throws IOException {
     File newFile = File.createTempFile(file, "test");
     InputStream in = QueueTestUtils.class.getResourceAsStream(file);
-    FileUtils.copyInputStreamToFile(in, newFile);
+    try (BufferedSink sink = Okio.buffer(Okio.sink(newFile))) {
```

Comment:
Realize I ported in this later change to use Okio for writing the file in the tests. Should be OK as this is TestUtils and won't effect production usages.

Reply:
Removed it anyway to avoid unnecessary changes.